src/typings/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.png" {
```

Comment:
What is it for?

Reply:
To let typescript know that `.png` files exist and treat them as modules. You can see import from png file in the AnimatedPlanet.tsx — it gets transformed into path at the build stage.